src/main/java/de/zalando/ep/zalenium/container/kubernetes/KubernetesContainerClient.java

```diff
@@ -610,16 +639,24 @@ public static DoneablePod createDoneablePodDefaultImpl(PodConfiguration config)
                         .endResources()
                         // Add a readiness health check so that we can know when the selenium pod is ready to accept requests
                         // so then we can initiate a registration.
+                        .withNewLivenessProbe()
```

Comment:
Why do we need a liveness probe?

Reply:
You're right, we don't need the livenessprobe, we'll also remove it.